tasks/prime.py

```diff
@@ -324,6 +324,12 @@ def create_mto_service_item(self, overrides=None):
     @tag(MTO_SHIPMENT, "createMTOShipment")
     @task
     def create_mto_shipment(self, overrides=None):
+        def guarantee_unique_agent_type(agents):
+            agent_types = {agent["agentType"] for agent in agents}
```

Comment:
Nice, I realized you were doing a set operation below but missed at first that this coerces duplicates.